assets/js/blocks/reviews/frontend-block.tsx

```diff
@@ -2,23 +2,43 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import PropTypes from 'prop-types';
 import { getSetting } from '@woocommerce/settings';
 import LoadMoreButton from '@woocommerce/base-components/load-more-button';
 import {
 	ReviewList,
 	ReviewSortSelect,
 } from '@woocommerce/base-components/reviews';
 import withReviews from '@woocommerce/base-hocs/with-reviews';
+import type { ChangeEventHandler, MouseEventHandler } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { ReviewListAttributes } from '../../base/components/reviews/review-list';
+import { Review } from '../../base/components/reviews/types';
+
+type FrontendBlockAttributes = ReviewListAttributes & {
+	showOrderby?: 'false' | 'true';
+	showLoadMore?: 'false' | 'true';
+};
+
+interface FrontendBlockProps {
+	attributes: FrontendBlockAttributes;
+	onAppendReviews: MouseEventHandler;
+	onChangeOrderby: ChangeEventHandler< HTMLSelectElement >;
+	sortSelectValue: 'most-recent' | 'highest-rating' | 'lowest-rating';
+	reviews: Review[];
+	totalReviews: number;
+}
 
 /**
  * Block rendered in the frontend.
  *
- * @param {Object}                                             props                 Incoming props for the component.
- * @param {Object}                                             props.attributes      Incoming block attributes.
- * @param {function(any):any}                                  props.onAppendReviews Function called when appending review.
- * @param {function(any):any}                                  props.onChangeOrderby
- * @param {Array}                                              props.reviews
+ * @param {FrontendBlockProps}                                 props                 Incoming props for the component.
```

Comment:
I cleaned up the JS doc but looking around the repo, I'm not sure if anyone is using JS Doc right now?

Reply:
Thanks, removed in [6248528](https://github.com/woocommerce/woocommerce-blocks/pull/10677/commits/6248528985fde1900a1631cb7c74fe00b4eb6f2e)